app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Login Page</h1>
+
+
+
+<%= form_tag sessions_path do %>
```

Comment:
You may want to consider using the newer Rails helpers like `form_with`, see here https://guides.rubyonrails.org/form_helpers.html.

`form_tag` is an older method and deprecated, which means it might be removed in future versions of Rails

Reply:
Thanks, I changed it.